### ISSUE TITLE
Multiple Monitors

### ISSUE THREAD

**Issue by Aaahh** [wacossusca34/glava#40]
Glava only runs on the main monitor, which is expected, but it would be nice if it ran on both. How to get glava to run on both monitors is beyond me. I don't know the proper way that actually works but glava window could be re-positioned onto the second monitor and then expanded (full screen, whatever).

**Comment by wacossusca34** [comment]
@Aaahh Do you mean separate X screens? Two monitors that are responsible for displaying the same screen area can certainly both run GLava (simply offset screen coordinates by an entire monitor, depends on X configuration for actual coordinates). You should be able to drag a normal GLava window across monitors, too.

Two separate X screens might run into issues, although not a lot of people tend to run these kinds of setups. `"xroot"` style transparency also breaks when GLava is not placed completely within the working space of a monitor (hanging off screen), but `"native"` transparency should be used instead anyway.

Example (GLava window is displayed partially on the bottom and top monitor): https://i.imgur.com/RUbsPjt.png, separate instances on each monitor also work.

**Comment by wacossusca34** [comment]
For reference, my regular `rc.glsl` contains the following (note the `setgeometry` line) to position GLava windows across monitors:

```GLSL

#request mod bars

#request setfloating  false
#request setdecorated false
#request setfocused   false
#request setmaximized false

#request setgeometry 17 1722 1886 380
#request setforcegeometry true

#request setopacity "native"

#request settitle "GLava"

#request setxwintype "desktop"

#request setsource "auto"

#request setswap 1

#request setinterpolate true

#request setframerate 0
#request setprintframes true

#request setsamplesize 1024
#request setbufsize 4096
#request setsamplerate 22050
#request setbufscale 1

#request setversion 3 3
#request setshaderversion 330
```

**Comment by wacossusca34** [comment]
Closing, since I'm pretty sure this is a non-issue.

**Comment by RaitaroH** [comment]
@wacossusca34 I would like to mention that I put glava on the left side on the screen. After I added a 2nd monitor glava now took half the screen for some reason. So I had to do: 

`#request setgeometry 0 -1080 1920 2160`

So it seems that because my resolution doubled from 1080 I had to put there 2160 but had to also move it up for some reason. This happens in plasma even when one of the screens is off so ¯\_(ツ)_/¯ 

Not much of a bug just something to mention for people who might face it.
